code/game/objects/items/weapons/storage/bible.dm

```diff
@@ -32,3 +32,82 @@
 	if (src.use_sound)
 		playsound(src.loc, src.use_sound, 50, 1, -5)
 	..()
+
+/obj/item/weapon/storage/bible/proc/Set_Religion(mob/user)
+	if(use_check(user))
+		return
+
+	if(!ishuman(user))
+		return
+
+	var/religion_name = "Christianity"
+	var/new_religion = sanitize(input(user, "You are the crew services officer. Would you like to change your religion? Default is Christianity, in SPACE.", "Name change", religion_name), MAX_NAME_LEN)
+
+	if (!new_religion)
+		new_religion = religion_name
+
+	var/deity_name = "Space Jesus"
+	var/new_deity = sanitize(input(user, "Would you like to change your deity? Default is Space Jesus.", "Name change", deity_name), MAX_NAME_LEN)
+
+	if ((length(new_deity) == 0) || (new_deity == "Space Jesus") )
+		new_deity = deity_name
+
+	deity_name = new_deity
+	SSticker.Bible_deity_name = new_deity
+
+	var/book_name = "holy bible"
+	book_name = sanitize(input(user, "Would you like the change your bible name? Default is holy bible.", "Book name change", book_name), MAX_NAME_LEN)
+
+	if (!book_name)
+		book_name = book_name
+
+	name = book_name
+	SSticker.Bible_name = book_name
+
+
+	var/new_book_style = input(user,"Which bible style would you like?") in list("Bible", "Koran", "Scrapbook", "Creeper", "White Bible", "Holy Light", "Atheist", "Tome", "The King in Yellow", "Ithaqua", "Scientology", "the bible melts", "Necronomicon")
+	switch(new_book_style)

```

Comment:
Why not give each religion datum a distinct book type and remove this type of selection?

Reply:
This selection only happens if the chaplain did not pick one of the lore religions, so they can create their own snowflake stuff.

Reply:
hmmm. oke.